apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -167,6 +184,87 @@ function LandingPage({
     return <ActionBlocksWrapper actionBlocks={actionBlocks} />;
   };
 
+  const RenderOwnedPlSections = () => {
+    return (
+      <section>
+        <Heading2>{i18n.plSectionsInstructorTitle()}</Heading2>
+        <SetUpSections
+          headingText={i18n.newSectionCreate()}
+          descriptionText={i18n.newSectionMyPlAdd()}
+          solidBorder={true}
+        />
+        <CoteacherInviteNotification isForPl={true} />
+        <OwnedSections
+          isPlSections={true}
+          sectionIds={plSectionIds}
+          hiddenSectionIds={hiddenPlSectionIds}
+        />
+        <AddSectionDialog />
+      </section>
+    );
+  };
+
+  const RenderMyPlTab = () => {
+    return (
+      <>
+        {showGettingStartedBanner && RenderGettingStartedBanner()}
+        {lastWorkshopSurveyUrl && RenderLastWorkshopSurveyBanner()}
+        {plCoursesStarted?.length >= 1 && RenderSelfPacedPL()}
+        <JoinSectionArea
+          initialJoinedStudentSections={joinedStudentSections}
+          initialJoinedPlSections={joinedPlSections}
+          isTeacher={true}
+          isPlSections={true}
+        />
+        {RenderEnrolledWorkshops()}
+        {deeperLearningCourseData?.length >= 1 && (
+          <section>
+            <Heading2>Online Professional Learning Courses</Heading2>
+            <ProfessionalLearningCourseProgress
+              deeperLearningCourseData={deeperLearningCourseData}
+            />
+          </section>
+        )}
+        <section>
+          <Heading2>{i18n.plLandingRecommendedHeading()}</Heading2>
+          {RenderStaticRecommendedPL()}
+        </section>
+      </>
+    );
+  };
+
+  const RenderFacilitatorCenterTab = () => {
+    return (
+      <>
+        {lastWorkshopSurveyUrl && RenderLastWorkshopSurveyBanner()}
+        {RenderOwnedPlSections()}
+        {RenderEnrolledWorkshops()}
```

Comment:
For facilitators, regional partners, and workshop organizers, we want to render the workshops related to that role. That data should be passed down to the client, but you might need to pull it through to this component.

[Here's](https://github.com/code-dot-org/code-dot-org/blob/105a338f60c14e399e5febca6c647cfd3a4749f6/dashboard/app/controllers/pd/professional_learning_landing_controller.rb#L24-L31) where that data is fetched in the controller.

Reply:
Oh you're so right, my bad! I'll fix this!